test/run_ci.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+if [ -z "${PROJECT_ID}" ]; then
+	echo "The PROJECT_ID ENV variable must be set to proceed. Aborting."
+	exit 1
+fi
+
+if [ -z "${SERVICE_ACCOUNT_JSON}" ]; then
+	echo "The SERVICE_ACCOUNT_JSON ENV variable must be set to proceed. Aborting."
+	exit 1
+fi
+
+export TF_VAR_project_id="${PROJECT_ID}"
+export TF_VAR_region="${REGION:-us-east4}"
+export TF_VAR_zone="${ZONE:-us-east4-b}"
```

Comment:
These variables don't match those in `test/fixtures/shared/variables.tf`.

Reply:
Ok, thank you, fixed